components/root/root.jsx

```diff
@@ -121,6 +123,9 @@ export default class Root extends React.PureComponent {
 
         setSystemEmojis(EmojiIndicesByAlias);
 
+        this.props.actions.setBrowserNotificationsPermission();
+        this.props.actions.scheduleNextNotificationsPermissionRequest();
```

Comment:
Let me know how that discussion goes, and feel free to message me if you have any additional questions about how that should work. I've been thinking about that sort of thing recently since I've been brainstorming for a long term vision for the architecture of the web app, and both notifications and "where does the announcement bar live?" are things that I've been thinking about a lot lately.

Reply:
So, I have got couple of things so far.

1. The problem of bar overlapping the "Logout" button on the "Select team" (found by @matthewbirtch ) screen is not related exclusively to this particular bar. For instance, I have reproduced the same bug for "Site settings" bar as shown below:
![another-bar-overlap](https://user-images.githubusercontent.com/7871134/141645258-ef01e5c6-3b8c-4f63-b80b-4c82cfd3b9cf.jpg) I don't see any evidence that my CSS changes could lead to this appearance, that's why I assume that webapp has had this before. Should we create a separate ticket for fixing this universally for all bars?

2. It is not clear for me when it is appropriate to show the bar. I have introduced a simple fix which limits the bar display for team view only but I am quite hesitant about it. Does it make sense to show this bar in admin console as well? Are there any other app views, which I am unaware of, which could benefit from displaying this bar?

@hmhealey @esethna

Reply:
1. Yeah, let's make that a separate ticket for that. We've got a big layout reorganization coming up, so this might not get fixed before that, but it'll be good to make a note of at least.
2. You could get notifications in the system console, but I don't think that's necessary since users would have to go through the other parts of the app to get there. I'll defer to @esethna on that though